src/test/java/com/ams/restapi/attendance/AttendanceControllerTests.java

```diff
@@ -42,4 +43,42 @@ void shouldReturnCorrectPageSize() throws Exception {
                 .andExpect(jsonPath("$[*].type").value(everyItem(equalTo("LEFT"))))
                 .andDo(print());
     }
+
+    @Test
+    @WithMockUser(roles="INSTRUCTOR")
+     void shouldUpdateAttendenceRecord() throws Exception {
+        mockMvc.perform(put("/attendance/33")
```

Comment:
Add `import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf`

And then add `.with(csrf())` after the `perform(put` here